platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/SyncGossip.java

```diff
@@ -193,9 +193,9 @@ protected SyncGossip(
         final ThreadConfig threadConfig = platformContext.getConfiguration().getConfigData(ThreadConfig.class);
 
         final BasicConfig basicConfig = platformContext.getConfiguration().getConfigData(BasicConfig.class);
-
-        topology = new StaticTopology(random, addressBook, selfId, basicConfig.numConnections());
         final List<PeerInfo> peers = Utilities.createPeerInfoList(addressBook, selfId);
+
+        topology = new StaticTopology(random, peers, selfId, basicConfig.numConnections());
```

Comment:
I think probably the best thing to do is to replace the topology implementation with a simpler one that has no random graph. and we keep the current one for the future when we decide we want to support it

Reply:
done. Resolving.